src/vaishnaviVJun20/Ex2.java

```diff
@@ -0,0 +1,16 @@
+package vaishnaviVJun20;
+
+public class Ex2 {
+	
+	void Display()
```

Comment:
method name should be start with small letter.

Reply:
Done